integration/eth2network/eth2_configs.go

```diff
@@ -0,0 +1,175 @@
+package eth2network
+
+import (
+	"encoding/json"
+)
+
+const gethPreloadJSONScript = `
+key = "2e0834786285daccd064ca17f1654f67b4aef298acbb82cef9ec422fb4975622";
+pwd = "password";
+
+personal.importRawKey(key, pwd);
+accts = personal.listAccounts;
+
+personal.unlockAccount(accts[0], pwd, 0);
+console.log("Unlocked account");
+miner.setEtherbase(accts[0]);
+
+miner.start();
+console.log("Miner Started");
+`
+
+const beaconConfig = `CONFIG_NAME: interop
+PRESET_BASE: interop
+
+# Genesis
+GENESIS_FORK_VERSION: 0x20000089
+
+# Altair
+ALTAIR_FORK_EPOCH: 2
+ALTAIR_FORK_VERSION: 0x20000090
+
+# Merge
+BELLATRIX_FORK_EPOCH: 4
+BELLATRIX_FORK_VERSION: 0x20000091
+TERMINAL_TOTAL_DIFFICULTY: 2
+
+# Time parameters
+SECONDS_PER_SLOT: 6
+SLOTS_PER_EPOCH: 6
+
+# Deposit contract
+DEPOSIT_CONTRACT_ADDRESS: 0x4242424242424242424242424242424242424242
+`
+
+const baseGenesis = `{
```

Comment:
Is there a way we can pull out some of these constants so they're accessible from go? 

In particular I was thinking `DefaultChainID` and `PrefundedAddresses` would be useful.

Reply:
The DefaultChainID and the PrefundedAddresses are specified when starting the eth2network.
In: 
https://github.com/obscuronet/go-obscuro/pull/1050/files#diff-a71dc2c1219aeb7132c3bc95c0aea57e9431fce002eb39669d2ffb6e8080a03aR52-R61

The config values in the `eth2_configs.go` should never be used. But I see where you're coming from. The configs are there to enable the POS network. I'll add a comment specifying it.

Does that make sense ?

Reply:
Oh I see, yeah that make sense 👍